README.md

```diff
@@ -37,6 +37,10 @@ Do not run script in script folder, run it in the main folder!
 
 `python3 scraper/Script.py`
 
+## Data scraper test
+
+`DB_USER=postgres DB_PASSWORD=secret DB_HOST=localhost TOKEN=<your token> python3 -m unittest scraper/test.py -v`
```

Comment:
I'd prefer we force people to run against a local database when running these tests, since otherwise we risk cluttering our production database with test artifacts. Also it's faster to run against a local database and we can guarantee that it contains the same data every time.